tracing-opentelemetry/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 0.18.0 (September 18, 2022)
+
+### Breaking Changes
+
+- Upgrade to `v0.18.0` of `opentelemetry` ([#2303])
+  For list of breaking changes in OpenTelemetry, see the
+  [v0.18.0 changelog](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry/CHANGELOG.md#v0180).
+
+### Fixed
+
+- `on_event` respects event's explicit parent ([#2296])
+
+Thanks to @wprzytula for contributing to this release!
```

Comment:
it looks like there are no Markdown link references for the links to the PR numbers, so those won't render correctly on GitHub. Let's add them:
```suggestion
### Breaking Changes

- Upgrade to `v0.18.0` of `opentelemetry` ([#2303])
  For list of breaking changes in OpenTelemetry, see the
  [v0.18.0 changelog](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry/CHANGELOG.md#v0180).

### Fixed

- `on_event` respects event's explicit parent ([#2296])

Thanks to @wprzytula for contributing to this release!

[#2303]: https://github.com/tokio-rs/tracing/pull/2303
[#2296]: https://github.com/tokio-rs/tracing/pull/2296
```